src/connectors/oxigenio.fm.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+Connector.playerSelector = '.wp-lunaradio.lunaaudioplayer';
+
+Connector.artistTrackSelector = 'div[id^="wp"][id$="texttitle"] span:nth-child(1)';
+
+Connector.playButtonSelector = 'div[id^="wp"][id$="buttonplay"]';
+
+Connector.isStateChangeAllowed = () => {
+	const artistTrack = Util.getTextFromSelectors(Connector.artistTrackSelector);
+	return artistTrack.includes('-') && !artistTrack.match(/^\s*OXIGENIO - /);
+};
+
+Connector.onReady = Connector.onStateChanged;
```

Comment:
I think we can drop this one

Reply:
Sorry, could you explain in detail what you mean?

Reply:
My understanding of this onReady behaviour is when you need to override the default behaviour when the scripts  are injected.

See: https://github.com/inverse/web-scrobbler/blob/882fdba3702795d0a337a27323523cd81c147263/src/core/content/connector.js#L418

by default the state transitions are handled via the `playerSelector` which is watched for mutations.

Reply:
Okay, thanks for this explanation! I wasn't sure what's that line for, tbh.
I'll remove that, check if it works without it and if so, commit.

Reply:
Yeah I think some other connectors also used this pattern when in reality they don't need to. Aside that the additions looks good. Thanks for your support 🙏🏼